src/handlers/event.ts

```diff
@@ -85,20 +88,30 @@ export async function handleEvent(definition: EventHandler, request: RawRequest,
     return handleRawEvent(definition, request, context)
 }
 
+export async function createLambdaContext(
+    definition: EventHandler<never, never>,
+    context: Context,
+    services: Promise<never> | undefined,
+    config: Promise<never> | undefined
+): Promise<LambdaContext<never, never>> {
+    return {
+        logger: globalLogger,
+        metrics: globalMetrics,
+        tracer: globalTracer,
+        isSensitive: definition.isSensitive ?? false,
+        requestId: (requestIdGenerator === 'uuid' ? randomUUID : randomUUID)(),
```

Comment:
I think it may be better to go with `requestTraceId`, since this one clashes with `awsRequestId` and `requestId` on the APIGW event